### PR TITLE
Adds Developer Information to README

### PR DESCRIPTION
**Pull request by jermnelson**
Check list items 1 and 2 in #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,34 @@
...
+## Streamlit SPOC Verifier Application
 
-### Abstract Generator
+## Development
+We recommend using a Python virtual environment to install SPOC's package dependencies. SPOC is 
+currently developed with [Python 3.9](https://www.python.org/downloads/) and above. 
 
-### Paper Classifier
+### Setup
+1. Create a new virtual environment `python -m venv spoc-env`
```
Comment (jgreben) [inline README.md:13]: On my system I had to use `python3` instead of `python`.

File: README.md [hunk 2]
```diff
@@ -1,9 +1,34 @@
...
+1. Activate (on POSIX systems) `source spoc-env/bin/activate`
+1. Clone the current SPOC Git repository `git clone https://github.com/sul-dlss-labs/spoc.git`
+1. Enter directory and install dependencies, `cd spoc && pip install -r requirements.txt`
```
Comment (jgreben) [inline README.md:16]: Before this I had to run `python3 -m pip install --upgrade pip`

Comment (jermnelson) [reply README.md:16]: 👍 just updated to use `python3`

File: README.md (hunk 1 above)
Comment (jermnelson) [reply README.md]: 👍 Just updated to use `python3`

**Review by jgreben — APPROVED** [review]
LGTM

**Pull request merged**